.github/workflows/build.yml

```
@@ -270,7 +270,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-24.04]
-        openssl_ver: [3.0.16, 3.1.8, 3.2.4, 3.3.3, 3.4.1]
+        openssl_ver: [3.0.17, 3.1.8, 3.2.5, 3.3.4, 3.4.2, 3.5.2]
```

Maybe 3.1 should be dropped here?
```suggestion
        openssl_ver: [3.0.17, 3.2.5, 3.3.4, 3.4.2, 3.5.2]
```

---

Add one, drop one :)

OpenSSL 3.1 was EOL on π.2025:

* https://openssl-library.org/roadmap/index.html
* https://endoflife.date/openssl